Solutions/Corvus.Sql.Tenancy/Corvus/Sql/Tenancy/Internal/TenancySqlServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// <copyright file="TenancySqlServiceCollectionExtensions.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Sql.Tenancy.Internal
+{
+    using System;
+    using Corvus.Tenancy;
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// Adds installer methods for tenanted storage-related components.
+    /// </summary>
+    internal static class TenancySqlServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Add components for constructing tenant-specific blob storage containers.
```

Comment:
Eh?

Before the merge I think this said "SQL containers", which I believe was correct, but this most recent commit has changed it to "blob storage". Am I misreading this, or was this a merge mishap?

Reply:
Gah.

Reply:
Now, that's weird. I'm not sure how that happened.

Reply:
Copy and paste happened.